xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESound.h

```diff
@@ -59,6 +61,7 @@ class CActiveAESound : public IAESound
   static int64_t Seek(void *h, int64_t pos, int whence);
 
 protected:
+  CActiveAE *m_activeAE;
```

Comment:
I guess @Paxxi has used to wrong words right?
Right @FernetMenta AE is one of the best codebases around in Kodi and so far I know @Paxxi only tries to improve our overall architecture.